src/model/code-visualizer.ts

```diff
@@ -32,9 +32,7 @@ export class CodeVisualizer {
         currentLineWords.push(wordObj);
       }
     }
-    if (currentLineWords.length > 0) {
-      lines.push(currentLineWords);
-    }
+    lines.push(currentLineWords);
```

Comment:
Well, this should also be changed in the `getCodeLinewords` (or something similar) method:
It can now return `lines[linenumber - 1]` (you're counting the lines 1-based, right?)